src/sinol_make/util.py

```diff
@@ -316,6 +317,14 @@ def stringify_keys(d):
         return d
 
 
+def get_file_md5(file):
+    """
+    Function to get the md5 hash of a file.
+    """
```

Comment:
The function name already describes the functionality. The comment doesn't add anything new for the reader. Instead of making a comment, maybe the function name or argument names can be further improved? What do you think of `md5_hash_of_file(path)`?

Reply:
`_get_md5sums_path` is not a great name. Is the function calculating the md5sums of the path? Is there a need to make the function "private" by naming it with an underscore?

Reply:
I changed the name, but maybe it could be improved further

Reply:
Maybe. Can you try to come up with some and write them here?

Reply:
Maybe `get_file_md5sum_path` or `get_path_md5sum_file`?

Reply:
Or maybe just delete this whole function? It just calculates md5 sum of a file using other function and returns a path with this sum

Reply:
Sure, though remember that functions are often a replacement for comments -- a good function name describes what the code does. If this code is not immediately readable for the user, after removing the function consider adding some comments.

Reply:
How is it now?